### PR TITLE
Bug/issue 53

### LINKED ISSUE #53: In notUpdated rule should not be created with values 0 or negative in interval field
Right now a notUpdated rule can be created with interval = 0 or negative. Therefore each 0 sec Perseo verifies against CB for know if the attribute is updated. Overloading perseo with multiple requests to CB. Other problem is each request write in log, so it can fill up and stop working perseo.

esample:
```
{
    "VR":{
        "active":1,
        "cards":[
            {
                "sensorData":{
                    "dataType":"",
                    "measureName":"temperature"
                },
                "configData":{
                },
                "sensorCardType":"notUpdated",
                "conditionList":[
                    {
                        "Not":false,
                        "scope":"LAST_MEASURE",
                        "parameterValue":"0",
                        "operator":"EQUAL_TO",
                        "parameterName":""
                    }
                ],
                "timeData":{
                    "interval":"0",
                    "context":"",
                    "repeat":"-1"
                },
                "type":"SensorCard",
                "id":"card_1",
                "connectedTo":[
                    "card_2"
                ]
            }
        ],
        "name":"test1200",
        "service":"/my_service",
        "tenant":"my_tenant"
    },
    "name":"test1200",
    "action":{
    },
    "service":"/my_service",
    "tenant":"my_tenant",
    "nosignal":{
        "checkInterval":"0",
        "attribute":"temperature",
        "reportInterval":"0",
        "id":null,
        "idRegexp":null,
        "type":null
    },
    "_id":ObjectId("54ddd89ed306f25859eae96b")
}
```

### PR DESCRIPTION
**Pull request by crbrox**
Fixes #53

@fgalan @dmoranj 

FYI  @iariasleon

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -8,4 +8,5 @@ Do not allow 'id' and 'type' as attribute names
 Change parameter for POST action from 'URL' to 'url'
 Fix error in VR for updateAction (#55)
 Fix validation rule name (#44, #15)
+Fix negative and zero chek intervals for NSR (#53)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: chek -> check

In additon I'd include the expansion of NSR (not sure right now what that acronym means :)

Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Fixed in c9e8840

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete branch)

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**